docs/api.md

```diff
@@ -846,6 +847,44 @@ The object for the **services** section can be:
 * **dns**: a list of dns service with the ip address of a dns server
 * **dnsFromIPPool**: the IPPool from which to fetch the dns servers list
 
+#### Updating metaData and networkData
+
+The data template parts containing the metadata and networkData must be
+immutable since the BareMetalHost references the secrets and they are used
+at provisioning time, the secrets cannot be updated to be able to reprovision
+the node in the exact same state. This means that updates have to be done by
+creating a new template and referencing it in the new/updated
+Metal3MachineTemplate.
+
+The process to allow updating the metaData and networkData is then to create a
+new Metal3DataTemplate and reference the new one in the Metal3MachineTemplate.
+This requires the Metal3Data to be linked to both Metal3DataTemplate. This is
```

Comment:
both should refer to two things. What is the other one ?
may be **to both Metal3DataTemplates** ?

Reply:
Thanks corrected